recipes/parmed/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "ParmEd" %}
+{% set version = "2.7.3" %}
+{% set md5 = "027cd3b738318c044164ff5ca06295e5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: ParmEd-{{ version }}.tar.gz
+  url: https://github.com/{{ name }}/{{ name }}/archive/{{ version }}.tar.gz
+  md5: {{ md5 }}
+
+
+build:
```

Comment:
This package has a command line interface as an entry point:

https://github.com/ParmEd/ParmEd/blob/master/setup.py#L17

please add the appropriate entry:

https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#python-entry-points